runtime/near-vm-logic/src/logic.rs

```diff
@@ -2533,7 +2531,7 @@ impl<'a> VMLogic<'a> {
     }
 }
 
-#[derive(PartialEq, Serialize, Deserialize, Clone)]
+#[derive(Serialize, Deserialize, Clone)]
```

Comment:
Why do we need to remove `PartialEq`? Seems proper to compare outcomes comparing profiling as well. Is this because all tests compare vm outcomes directly? I wonder if there's a better solution here...

If the isn't, I am fine with the current approach, but we should leave a comment explaining why we do it this way. 

I think we should clean this up, and the Serialize as well, but that should be left for a follow up.

Reply:
In practice it's impossible that total used gas and burnt gas equals but profile data doesn't matches. Unless profile data is not working at all, but that would be caught by another test. So just comparing fields except profile data makes current tests that compare VMOutcome cleaner.

Reply:
I'll leave a code comment, since no matter we clean up it or not it'll be in a follow up.